### DISCUSSION TITLE
(answered) codespace not working

### DISCUSSION THREAD

**Discussion by Anonymous6-source** [UseInterstellar/Interstellar#889] [Q&A] [answered]
why does the codespace I made wont let me open it

**Comment by 0mhx** [comment]
send screenshots

**Reply by Anonymous6-source** [reply]
ok

**Reply by Anonymous6-source** [reply]
![image](https://github.com/user-attachments/assets/bc9158ed-f2d6-4d1f-a4e4-cf43878ac145)

**Reply by Anonymous6-source** [reply]
im new to this so i dont know anything

**Reply by ewarggg776** [reply]
do you have paid or free?

**Reply by Anonymous6-source** [reply]
free

**Comment by Anonymous6-source** [comment]
i already figured it out

**Comment by Anonymous6-source** [comment, marked as answer]
i justr needed to wait until it renewed
